This patch improves the stability of the mysql regression tests on Red Hat's
build servers.  I really have no idea why ... it is probably masking a
bug in mysqld, but without a better handle on how to reproduce the problem,
it's tough to make a bug report.  Worth noting is that rpl000001 passes
fine when run by itself, or with the immediately preceding tests; but fails
when the complete test series is run.


diff -Naur mysql-5.0.18.orig/mysql-test/r/rpl000001.result mysql-5.0.18/mysql-test/r/rpl000001.result
--- mysql-5.0.18.orig/mysql-test/r/rpl000001.result	2005-12-21 14:50:25.000000000 -0500
+++ mysql-5.0.18/mysql-test/r/rpl000001.result	2006-01-05 01:57:17.000000000 -0500
@@ -1,3 +1,4 @@
+flush privileges;
 stop slave;
 drop table if exists t1,t2,t3,t4,t5,t6,t7,t8,t9;
 reset master;
diff -Naur mysql-5.0.18.orig/mysql-test/t/rpl000001.test mysql-5.0.18/mysql-test/t/rpl000001.test
--- mysql-5.0.18.orig/mysql-test/t/rpl000001.test	2005-12-21 14:50:25.000000000 -0500
+++ mysql-5.0.18/mysql-test/t/rpl000001.test	2006-01-05 01:57:15.000000000 -0500
@@ -1,3 +1,4 @@
+flush privileges;
 source include/master-slave.inc;
 
 create table t1 (word char(20) not null);
